### PR TITLE
Show all issues on lint

### PR DESCRIPTION
**Pull request by otherview**
# Description

It looks like the thor client merge got away with passing some linter issues to the master code.
This mainly happened because the `only-new-issues` was active.
The fix is to guarantee the `only-new-issues` is deactivated.

Fixes # [(issue)](https://github.com/vechain/otherviews-workboard/issues/106)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] New and existing E2E tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have not added any vulnerable dependencies to my code

### CODE REVIEW

File: .github/workflows/lint-go.yaml [hunk 1]
```diff
@@ -23,5 +23,5 @@ jobs:
           version: v1.60.3
           # use the default if on main branch, otherwise use the pull request config
           args: --timeout=30m --config=.golangci.yml
-          only-new-issues: true
+          only-new-issues: false
```
Comment (darrenvechain) [inline .github/workflows/lint-go.yaml:26]:
```suggestion
          only-new-issues: ${{ github.event_name == 'pull_request' }}
```

File: api/accounts/accounts_test.go [hunk 2]
```diff
@@ -578,45 +576,3 @@ func batchCallWithNonExistingRevision(t *testing.T) {
 	assert.Equal(t, http.StatusBadRequest, statusCode, "bad revision")
 	assert.Equal(t, "revision: leveldb: not found\n", string(res), "revision not found")
 }
-
-func httpPost(t *testing.T, url string, body interface{}) ([]byte, int) {
```
Comment (leszek-vechain) [inline api/accounts/accounts_test.go:582]: why we remove all these methods ? are they somewhere else or we don't use them at all ?

File: .github/workflows/lint-go.yaml (hunk 1 above)
Comment (otherview) [reply .github/workflows/lint-go.yaml:26]: This seems to assume that it's expected only to detect new issues in pull requests, which I believe may fail in certain edge cases like the scenario that brought us here.
While there's a benefit of only running the lint for the diffs, golangci-lint linter is so fast that the difference looks negligible ?

File: api/accounts/accounts_test.go (hunk 2 above)
Comment (otherview) [reply api/accounts/accounts_test.go:582]: They're not used any longer. I think they were made redundant by the thor client merge.

**Review by darrenvechain — APPROVED** [review]

**Pull request merged**